batch/data-plane/src/test/java/com/microsoft/azure/batch/BatchTestBase.java

```diff
@@ -54,8 +54,8 @@ static CloudPool createIfNotExistPaaSPool(String poolId) throws Exception {
         String POOL_OS_FAMILY = "4";
         String POOL_OS_VERSION = "*";
 
-        // 5 minutes
-        long POOL_STEADY_TIMEOUT_IN_SECONDS = 5 * 60 * 1000;
+        // 10 minutes
+        long POOL_STEADY_TIMEOUT_IN_SECONDS = 10 * 60 * 1000;
```

Comment:
The variable name is 'in seconds', but the value is in milliseconds. It doesn't really matter to this PR (as the issue existed beforehand) - but perhaps it should be renamed to 'IN_MILLISECONDS'?